README.md

```diff
@@ -28,6 +28,10 @@ module "eks_cluster" {
 
   cluster_service_ipv4_cidr = "10.190.0.0/16"
   cluster_node_ipv4_cidr    = "10.192.0.0/16"
+
+  output "cluster_region" {
+    value = "eu-central-1"
+  }
```

Comment:
Not entirely sure what's the purpose of this.
It's wrapped in the module and not gonna work.

Also why pass the region and then pull the regional again?